README.md

```diff
@@ -1,40 +1,49 @@
-# inji-certify
-INJI Certify enables an issuer to connect with an existing database in order to issue verifiable credentials.
-It assumes the source database has a primary key for each data record and information required to authenticate a user (e.g. phone, email, or other personal information).
+# Inji Certify
+
+INJI Certify enables an issuer to connect with an existing Credential Registry to issue verifiable credentials.
 Issuer can configure their respective credential schema for various types of certificates they wish to issue. Certificates are generated in JSON-LD as per W3C VC v1.1.
 
-## Requirements
+# Requirements to run it locally (without docker)
+
 - Java SE 21
-- Maven 
-
-## Build
-```shell
-mvn clean install -Dgpg.skip=true
-```
-## Run
-```shell
-mvn spring-boot:run -Dspring.profiles.active=local
-```
+- Postgres
+- Maven
+- Redis, _if an Authorization provider like eSignet is also deployed_
```

Comment:
technically speaking redis is only needed if mock IDA plugin is used, should this even be mentioned under `requirements`?